### PR TITLE
feat(perf-issues): Prepare UI for Slow DB Span Issue Type

### PR DESCRIPTION
**Pull request by 0Calories**
Prepares the Span Evidence section for Slow DB Issue types, which will be coming in the near future. Also a bit of refactoring in the key value section to map the structure of the data to specific issue types.

### CODE REVIEW

**Review by gggritso — APPROVED** [review]
LGTM! I think you should also update `static/app/views/organizationGroupDetails/header.tsx` and make sure the Alpha badge is there for this issue type

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx [hunk 1]
```diff
@@ -95,11 +95,6 @@ describe('SpanEvidenceKeyValueList', () => {
         screen.getByTestId('span-evidence-key-value-list.transaction-name')
       ).toHaveTextContent('/');
 
-      expect(screen.getByRole('cell', {name: 'Parent Span'})).toBeInTheDocument();
-      expect(
-        screen.getByTestId('span-evidence-key-value-list.parent-name')
-      ).toHaveTextContent('pageload');
-
```
[the remark this answers is not in the document]
Comment (0Calories) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx:102]: Oh, I just thought I would give you one less merge conflict to deal with while I was here 😂

[the remark this answers is not in the document]
Comment (0Calories) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx:102]: On hindsight probably wasn't the best idea to put it in this pr 🤷

File: static/app/components/events/interfaces/performance/spanEvidence.tsx [hunk 2]
```diff
@@ -30,6 +30,10 @@ function getEvidenceDescription(issueType: IssueType) {
     return t('Span Evidence identifies the repeating network spans.');
   }
 
+  if (issueType === IssueType.PERFORMANCE_SLOW_SPAN) {
+    return t('Span Evidence identifies the slow database span.');
+  }
+
```
[the remark this answers is not in the document]
Comment (0Calories) [reply static/app/components/events/interfaces/performance/spanEvidence.tsx:36]: oo nice, thanks for the heads up

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (gggritso) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx:102]: 😂 that's nice of you! Yeah, don't worry about that, I can deal with conflicts

**Pull request merged**